src/commands/locks/list.ts

```diff
@@ -0,0 +1,130 @@
+import * as MixFlags from '../../utils/flags'
+import * as ProjectsAPI from '../../mix/api/projects'
+import {DomainOption} from '../../utils/validations'
+import {MixClient, MixResponse, MixResult,  ProjectsLockListParams} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+
+const debug = makeDebug('mix:commands:locks:list')
+
+export default class LocksList extends MixCommand {
+  static description = `list project locks
+
+Use this command to get a list of all project locks.
+The list can be constrained using the project, organization
+and/or user flags.
+A project cannot be edited while it is locked.`
+
+  static examples = [
+    'list project locks',
+    '$ mix locks:list -P 249 -O 33 -U 32',
+  ]
+
+  static flags = {
+    json: MixFlags.jsonFlag,
+    project: {
+      ...MixFlags.projectWithDefaultFlag,
+      required: false,
+    },
+    user: MixFlags.userFlag,
+    organization: {
+      ...MixFlags.organizationWithDefaultFlag,
+      required: false,
+    },
+    ...MixFlags.limitOffsetSortFlags,
+    ...MixFlags.tableFlags({except: ['extended', 'sort']}),
+    yaml: MixFlags.yamlFlag,
+  }
+
+  get columns() {
+    debug('get columns()')
+    return {
+      lockId: {header: 'LockId'},
+      projectId: {header: 'ProjectId'},
+      email: {
+        header: 'LockOwnerName',
+        get: (locks: any) => locks.lockOwner.email,
+      },
+      id: {
+        header: 'LockUserId',
+        get: (locks: any) => locks.lockOwner.id,
+      },
+      createTime: {header: 'CreateTime'},
+      notes: {header: 'Notes'},
+    }
+  }
+
+  get domainOptions(): DomainOption[] {
+    debug('get domainOptions()')
+    return ['limit', 'offset', 'organization', 'project', 'user']
+  }
+
+  async buildRequestParameters(options: Partial<flags.Output>): Promise<ProjectsLockListParams> {
+    debug('buildRequestParameters()')
+    const {limit = defaultLimit, offset, organization: orgId, project: projectId, user: userId, sort: sortBy} = options
+
+    return {
+      ...(typeof limit === 'undefined' ? {} : {limit}),
+      ...(typeof offset === 'undefined' ? {} : {offset}),
+      ...(typeof orgId === 'undefined' ? {} : {orgId}),
+      ...(typeof projectId === 'undefined' ? {} : {projectId}),
+      ...(typeof userId === 'undefined' ? {} : {userId}),
+      ...(typeof sortBy === 'undefined' ? {} : {sortBy}),
+    }
+  }
+
+  doRequest(client: MixClient, params: ProjectsLockListParams): Promise<MixResponse> {
+    debug('doRequest()')
+    return ProjectsAPI.listProjectLock(client, params)
+  }
+
+  outputHumanReadable(transformedData: any) {
+    debug('outputHumanReadable()')
+    const {columns} = this
+
+    if (transformedData.length === 0) {
+      this.log('No locks found.')
+
+      return
+    }
+
+    super.outputCLITable(transformedData, columns)
+  }
+
+  setRequestActionMessage(options: any) {
+    debug('setRequestActionMessage()')
+    const project = options.project ? `project ${options.project}` : ''
```

Comment:
rename `project`, `user` and `organization` to `projectFilter`, `userFilter` and `organizationFilter`.